apps/dialtone-documentation/docs/.vuepress/exampleComponents/ExampleModal.vue

```diff
@@ -1,224 +1,84 @@
 <template>
-  <div class="d-d-flex d-ai-flex-end d-jc-space-between">
-    <div v-if="bannerTitle" class="d-w100p d-mr8">
-      <dt-select-menu
-        label="Kind of Banner"
-        size="sm"
-        @change="changeBannerKind"
-      >
-        <option
-          v-for="option in bannerKinds"
-          :key="option"
-          :selected="option === selectedBannerKind"
-          :value="option"
-          v-text="option"
-        />
-      </dt-select-menu>
-    </div>
-    <button
-      class="d-btn d-btn--primary d-btn--sm d-fl-none"
-      type="button"
-      @click="openModal"
-    >
-      Launch Example
-    </button>
-  </div>
-  <aside
-    id="modal-base"
-    ref="modal"
-    class="d-modal d-m0"
-    :class="{
-      'd-modal--full': isFullScreen,
-      'd-modal--danger': isDanger,
-      'd-modal--animate-in': animateIn,
-      'd-modal--animate-out': animateOut,
-    }"
-    tabindex="-1"
-    role="dialog"
-    aria-labelledby="modal-title"
-    aria-describedby="modal-description"
-    :aria-hidden="!showModal"
-    @click.self="closeModal"
-    @keydown.tab="trapFocus"
-    @keydown.esc="closeModal"
+  <dt-modal
+    title="Example title"
+    :close-button-props="{ ariaLabel: 'Close modal' }"
+    :show="isOpen"
+    :banner-title="bannerTitle"
+    :banner-kind="bannerKind"
+    :fixed-header-footer="fixedHeaderFooter"
+    :size="size"
+    :copy="modalDescription"
```

Comment:
Previously, in the case where "fixedHeaderFooter" prop was true, we had the copy repeated to show that the header and footer are fixed and only the content is scrollable. I think we should keep that.

Reply:
ah yeah good catch, I'll add that back.

Reply:
This one is fixed